src/libraries/Native/Unix/System.Native/pal_process.c

```diff
@@ -653,6 +653,12 @@ int32_t SystemNative_SetRLimit(RLimitResources resourceType, const RLimit* limit
 
 int32_t SystemNative_Kill(int32_t pid, int32_t signal)
```

Comment:
Maybe this can even be:

```
    switch (signal)
    {
        case PAL_SIGKILL:
             signal = SIGKILL;
             break;

        case PAL_SIGSTOP:
             signal = SIGSTOP;
             break;

        default:
             assert_msg(false, "Unknown signal");
             break;
    }
```

And delete `c_static_assert(PAL_SIGKILL == SIGKILL);` at the top of the file. I have verified that clang optimizes the switch out when the remap-signal numbers  match.

Reply:
Thanks!